OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -47,11 +46,20 @@ Start a container from the WebLogic install image. You can override the default
 
 NOTE: To set the DOMAIN_NAME, you must set both DOMAIN_NAME and DOMAIN_HOME.
 
-        $ docker run -d -e ADMIN_USERNAME=weblogic -e ADMIN_PASSWORD=welcome1 -e DOMAIN_HOME=/u01/oracle/user_projects/domains/abc_domain -e DOMAIN_NAME=abc_domain -p 7001:7001 store/oracle/weblogic:12.2.1.3
+        $ docker run -d -e ADMIN_USERNAME=weblogic \
+                     -e ADMIN_PASSWORD=welcome1 \
+                     -e DOMAIN_HOME=/u01/oracle/user_projects/domains/abc_domain \
+                     -e DOMAIN_NAME=abc_domain \
+                     -p 7001:7001 \
+                     store/oracle/weblogic:12.2.1.3
 
 Start an Apache container by calling:
 
-        $ docker run -d --name apache -e WEBLOGIC_HOST=<admin-host> -e WEBLOGIC_PORT=7001 -p 80:80 12213-apache
+        $ docker run -d --name apache \
+                     -e WEBLOGIC_HOST=<admin-host> \
+                     -e WEBLOGIC_PORT=7001 \
+                     -p 80:80 \
+                     12213-apache
```

Comment:
to distinguish this image as a oracle image maybe the buildDockerImages.sh should call this image oracle/apache:12.2.1.3 to be consistent with our other images.  I know our domain is 12213-domain but this image never will be public, the weblogic images that are public are named oracle/weblogic:12.2.1.3.

Reply:
store/oracle/weblogic:12.2.1.3?

Reply:
this is what we tag the image in kubernetes operator project.

Reply:
not store/oracle/apache:12.2.1.3 because the apache image is not in the Docker Store or OCR.  But when a customer builds it using buildDockerImages.sh the name could be oracle/apache:12.2.1.3.  If this is too disruptive then leaving it as is is fine.

Reply:
Okay. Let me see what I can do.

Reply:
Image name changed.